onnxruntime/test/onnx/main.cc

```diff
@@ -480,6 +480,48 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"bitshift_right_uint16", "BitShift(11) uint16 support not enabled currently"},
       {"bitshift_left_uint16", "BitShift(11) uint16 support not enabled currently"},
       {"maxunpool_export_with_output_shape", "Invalid output in ONNX test. See https://github.com/onnx/onnx/issues/2398"},
+      {"averagepool_2d_ceil", "shape mismatch", {"onnx150", "onnxtip"}},
+      {"batchnorm_epsilon_training_mode", "Training_mode is not a scalar boolean", {"onnxtip"}},
```

Comment:
These test failures should be fixed.

Reply:
they are not failing anymore. the latest version does not have them.